container/webui/nginx.conf

```diff
@@ -14,17 +14,18 @@ server {
   # ssl_certificate_key /etc/ssl/certs/openqa.key;
 
   location /api/v1/ws/ {
-      proxy_pass http://webui_websockets_1:9527;
+      proxy_pass http://websockets:9527;
       proxy_http_version 1.1;
       proxy_set_header Upgrade $http_upgrade;
       proxy_set_header Connection "upgrade";
       proxy_set_header Host $host;
-
+      proxy_read_timeout     300;
+      proxy_connect_timeout  300;
```

Comment:
This might make sense for the liveviewhandler as well although it should usually try to re-connect when running into a timeout.

Reply:
done